examples/Clients/Greeter/Program.cs

```diff
@@ -38,16 +38,29 @@ static async Task Main(string[] args)
             var reply = client.SayHello(new HelloRequest { Name = "GreeterClient" });
             Console.WriteLine("Greeting: " + reply.Message);
 
-            var replies = client.SayHellos(new HelloRequest { Name = "GreeterClient" });
-            while (await replies.ResponseStream.MoveNext(CancellationToken.None))
```

Comment:
The semantics of cancellation token passed to responseStream.MoveNext() is to trigger cancellation of the entire call - hence the RpcException being thrown, because it means the call actually ended with an error. Also, because the call is being terminated, it makes sense to throw  RpcException with corresponding status (that's the same behavior you'd get from a unary call - if you cancel it, it will throw RpcException once it get cancelled).